policy-supplemental/platforms.md

```diff
@@ -2,16 +2,17 @@
 
 | Target           | &nbsp; | O/S                 | &nbsp; | Architecture | &nbsp; | Toolchain                            |
 |------------------|--------|---------------------|--------|--------------|--------|--------------------------------------|
-| linux-x86\_64    |        | Ubuntu Server 20.04 |        | x86\_64      |        | gcc 9                                |
-| linux-generic64  |        | Ubuntu Server 20.04 |        | x86\_64      |        | gcc 9                                |
+| linux-x86\_64    |        | Ubuntu Server 22.04 |        | x86\_64      |        | gcc 11                               |
+| linux-generic64  |        | Ubuntu Server 22.04 |        | x86\_64      |        | gcc 11                               |
 | linux-x86        |        | Debian 11           |        | x86          |        | gcc 10                               |
 | linux-generic32  |        | Debian 11           |        | x86          |        | gcc 10                               |
```

Comment:
I would remove Debian lines as well which doesn't mean that it is not supported but mostly that we don't run CI against it.

Reply:
Doesn't this require a change to [the Platform Policy](https://www.openssl.org/policies/general/platform-policy.html)? Since it says

> **Primary**
> Definition: A platform that is regularly tested through project CI on a project owned and managed system.

But the project doesn't have its own AArch64 system.

(Don't get me wrong, this would be great, and we are happy to continue running our CI server)

Reply:
We have aarch64 instances in our GCP account and we run the CI against them https://github.com/openssl/openssl/blob/master/.github/workflows/ci.yml#L116